build.sh

```diff
@@ -2,17 +2,5 @@
 
 DOCKER_IMAGE=mjdymalla/iso3166-library
 
-if [ $1 = "rebuild" ]
-then
-    docker build --no-cache -t $DOCKER_IMAGE .
```

Comment:
It's probably ok to expect a developer to know if they want to force an image re-build, they should `docker rmi mjdymalla/iso3166-library`